src/ctia/graphql/schemas.clj

```diff
@@ -155,8 +160,14 @@
                               p/connection-arguments)
                  :resolve (res/search-entity-resolver :weakness)}}))
 
-(def schema (g/new-schema QueryType))
-(def graphql (g/new-graphql schema))
+(s/def schema :- (MaybeDelayedGraphQLValue GraphQLSchema)
+  (g/new-schema QueryType))
+(s/def graphql :- (MaybeDelayedGraphQLValue GraphQL)
+  (g/new-graphql schema))
```

Comment:
Part of the point of introducing delayed graphql values is so vars like the above don't need to be changed. They do, however, change their schema. There are dozens of vars that I didn't explicitly add `s/def`'s for because they're not "directly" used, just stuffed into a global map somewhere---it would probably be too noisy to add schemas.

I think the above `graphql` and `schema` values are important enough to annotate though, since they're used directly to as an entry point to building the GraphQL "graph".